core/domain/role_services_test.py

```diff
@@ -37,16 +37,16 @@ def test_get_role_actions_return_value_in_correct_schema(self):
             for action_name in allotted_actions:
                 self.assertTrue(isinstance(action_name, str))
 
-    def test_get_all_actions(self):
-        with self.assertRaisesRegex(
+    def test_get_all_actions(self) -> None:
+        with self.assertRaisesRegex(# type: ignore[no-untyped-call]
```

Comment:
Please leave a couple of spaces before the '#'.

Reply:
Made the require changes in the recent commit. @seanlip PTAL
Thanks